platforms/Cross/plugins/FilePlugin/FilePlugin.h

```diff
@@ -30,6 +31,7 @@ typedef int mode_t;
 typedef struct {
   int			 sessionID;	/* ikp: must be first */
   void			*file;
+  mode_t		st_mode;	/* from stat() */
```

Comment:
By the way, do we still need st_mode and setMode now?

Reply:
Yeah, I just saw that.  I don't know how I managed to lose part of the commit.  There's also code left behind in sqFileBasicPrims.c.

I'm fixing it now.  Sorry about that.

Reply:
No problem, there should be a special prime to whoever remove code!

Reply:
Done.  Hopefully I've got them all this time.